pkg/attestation/renderer/renderer.go

```diff
@@ -178,11 +178,15 @@ func addPolicyResults(statement *intoto.Statement, policyResults []*v1.PolicyEva
 		return fmt.Errorf("unmarshalling predicate: %w", err)
 	}
 
-	// insert policy evaluations
+	// insert policy evaluations for attestation
```

Comment:
I think there is smth wrong with this method, it still references `v1 "github.com/chainloop-dev/chainloop/pkg/attestation/crafter/api/attestation/v1"` and we might don't need this?

Reply:
It was needed by PolicyVerifier. I've moved all policy-related logic to v02, since that's the predicate version that understands about policies. Now `renderer.go` is clean.